bundles/org.openhab.binding.goecharger/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -158,6 +159,12 @@
 		<description>Power on L3</description>
 		<state pattern="%.1f %unit%" readOnly="true"/>
 	</channel-type>
+	<channel-type id="pAll">
```

Comment:
Please update the file goeacharger.properties.
This can be donbe automatically by simply running this command: mvn org.openhab.core.tools:i18n-maven-plugin:3.2.0:generate-default-translations